iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/match/voter/GetOrgAlternativeNamesFunction.java

```diff
@@ -0,0 +1,36 @@
+package eu.dnetlib.iis.wf.affmatching.match.voter;
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.function.Function;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableList;
+
+import eu.dnetlib.iis.wf.affmatching.model.AffMatchOrganization;
+
+/**
+ * Function that returns {@link AffMatchOrganization#getAlternativeNames()} of the passed organization.
+ * 
+* @author Łukasz Dumiszewski
+*/
+
+public class GetOrgAlternativeNamesFunction implements Function<AffMatchOrganization, List<String>>, Serializable {
```

Comment:
I assume this function was implemented in advance to be used in future, right? I don't see any reference to that function other than unit tests.